src/components/media-loader.js

```diff
@@ -42,7 +42,7 @@ AFRAME.registerComponent("media-loader", {
     if (this.el.body && this.shapeAdded && this.el.body.shapes.length > 1) {
       this.el.removeAttribute("shape");
       this.shapeAdded = false;
-    } else if (!this.hasBakedShapes) {
+    } else if (!this.hasBakedShapes && !box.isEmpty()) {
```

Comment:
these various `box.isEmpty()` checks are addressing a problem where if you paste a GLTF file that has nothing but `media-loader`, it fails on these code paths because the initial bounding box is empty